### PR TITLE
Log services

### LINKED ISSUE #319: Log error message, if initialization / recover fails
When calling the service interface to init / recover the motor, there is an error string in the response of the service call, if something fails. Calling the service from the launch file on the other hand does not give any error indication to the user.

There should be the same error string printed to the console:

```
[ INFO] [1549975063.533360109]: Initializing XXX
[ INFO] [1549975063.533741997]: Current state: 1 device error: system:0 internal_error: 0 (OK)
[ INFO] [1549975063.534281762]: Current state: 2 device error: system:0 internal_error: 0 (OK)
--> Error, but no warning / error message
```

### PR DESCRIPTION
**Pull request by mathias-luedtke**
closes #319, replaces #343 

@jschleicher: please give this a try

### CODE REVIEW

File: canopen_chain_node/src/ros_chain.cpp [hunk 1]
```diff
@@ -90,8 +90,48 @@ void RosChain::run(){
...
+    const T& res;
+    const std::string command;
+public:
+    ResponseLogger(const T& res, const std::string &command) : logged(false), res(res), command(command){}
+    ~ResponseLogger() {
+        if(!logged && !res.success){
+            if (res.message.empty()){
+                ROS_ERROR_STREAM(command << " failed");
+            }else{
+                ROS_ERROR_STREAM(command << " failed:" << res.message);
```
Comment (jschleicher) [inline canopen_chain_node/src/ros_chain.cpp]:
```suggestion
                ROS_ERROR_STREAM(command << " failed: " << res.message);
```

**Review by jschleicher — CHANGES_REQUESTED** [review]
Yep, works well.

Just a tiny nit-picking comment.

**Comment by mathias-luedtke** [comment]
Thanks!

I will rebase it later and merge.

**Review by jschleicher — APPROVED** [review]

**Comment by mathias-luedtke** [comment]
This introduced [a regression](https://travis-ci.org/ros-industrial/ros_canopen/jobs/592239901) for Debian stretch, mostly because of an outdated version of `console_bridge`, which defines a `logWarn` macro.

**Pull request merged**